src/OrchardCore/OrchardCore.Data.YesSql/Removing/ShellDbTablesInfo.cs

```diff
@@ -66,17 +67,30 @@ public IEnumerable<string> GetTableNames()
     }
 
     public ISchemaBuilder CreateMapIndexTable(Type indexType, Action<ICreateTableCommand> table, string collection)
+    {
+        CreateMapIndexTableAsync(indexType, table, collection).GetAwaiter().GetResult();
+        return this;
+    }
+
+    public Task CreateMapIndexTableAsync(Type indexType, Action<ICreateTableCommand> table, string collection)
```

Comment:
Why are these asyncs needed? It seems to me that none of them actually do any async work.

Reply:
We added these methods to  ISchemaBuiler interface in yessql. So we need to implement them here

Reply:
Ah yeah, I missed that there's an interface from YesSql.